domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -143,6 +143,7 @@ class StoryProgressTestHelper @Inject constructor(
     } else {
       getOldTimestamp()
     }
+    markFullTopicProgressForFractions(profileId, timestampOlderThanOneWeek)
```

Comment:
Based on the name of this function which is `markFullTopicProgressForTestTopics` marking of fractions topic as full progress seems incorrect to me.

Reply:
It was added by mistake. It is incorrect so removed it.